image/util.go

```diff
@@ -113,3 +113,18 @@ func GetClusterFileFromBaseImage(imageName string) string {
 	}
 	return string(data)
 }
+
+func GetImageByClusterFileName(imageName string) string {
+	image, err := imageUtils.GetImage(imageName)
```

Comment:
The name of function has problem, we want to get Clusterfile from image, NOT get image by ClusterfileName.

the content of this function is reduplicate with GetClusterFileFromImageManifest I think